spec/models/item_spec.rb

```diff
@@ -24,4 +30,28 @@
     end
   end
 
+  describe '.valid_items' do
+    it "returns all items that have not expired" do
+      expect(Item.valid_items).to include item
+    end
+
+    it "does not return expired items" do
+      expect(Item.valid_items).to_not include item2
+    end
+  end
+
+  describe "#expires_on" do
+    it "returns an expiration date for an item" do
+      expect(item.expires_on).to eq (Date.today + 14.days)
+      expect(item2.expires_on).to eq (Date.today - 1.days)
+    end
+  end
+
+  describe "#price_in_dollars" do
+    it "returns a dollar amount for price" do
+      expect(item.price_in_dollars).to eq '0.10'
+      expect(item2.price_in_dollars).to eq '0.03'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.